File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Wind.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Wind {
    @Json(name = "1h")
    double speed;

    @Json(name = "deg")
    int degree;

    @Json(name = "gust")
    String metrics;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Wind.kt
package com.riders.thelab.data.remote.dto.weather

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Wind(
    @Json(name = "1h")
    val speed: Double = 0.0,

    @Json(name = "deg")
    val degree: Int = 0,

    @Json(name = "gust")
    val metrics: String
)
